manifests/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: objectstorage-controller
+  namespace: objectstorage-system
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  selector:
+    matchLabels:
+      app: objectstorage-controller
```

Comment:
Let's be a bit more specific, e.g.:

```
app.kubernetes.io/part-of: cosi
app.kubernetes.io/name: cosi-controller-manager
app.kubernetes.io/component: controller
```
Then, have a `app.kubernetes.io/instance` in the root `Kustomization`, e.g. set to `cosi-controller-manager` (as well).

Again, see https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels.